tests/python/pants_test/backend/jvm/tasks/test_junit_run_integration.py

```diff
@@ -144,6 +144,11 @@ def test_junit_run_with_coverage_succeeds_cobertura_merged(self):
             args=["--batch-size=2"],
         )
 
+    def test_junit_run_with_coverage_succeeds_cobertura_output(self):
```

Comment:
If you are testing a change in the output of the pants you should probably assert something about the output.